### ISSUE TITLE
Can zigate read aqare gadgets attributes?

### ISSUE THREAD

**Issue by SchumyHao** [elric91/ZiGate#20]
Hi 
I found zigate update attributes value only when aqare gadgets report attribute value.
Can zigate read the attributes value directly?

**Comment by elric91** [comment]
Yes, commands are implemented in command_helpers (zigate.read_attributes, zigate.read_multiple_attributes)
The objects may not respond immediately, so there may be a delay between call and response

**Comment by ISO-B** [comment]
Also some objects wan't you to push button after receiving request.
